quizcon/templates/main/student_help.html

```diff
@@ -0,0 +1,93 @@
+{% load quiz_tools %}
+<div class="modal fade" id="studentHelp" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="studentHelpLabel" aria-hidden="true">
```

Comment:
I'm seeing that the modal grows & shrinks based on the content. Let's work to get the sizing consistent for all four "pages".